lib/sshkit/scp/command.ex

```diff
@@ -3,22 +3,29 @@ defmodule SSHKit.SCP.Command do
 
   import SSHKit.Utils
 
-  def build(mode, path, options) do
-    scp = case mode do
-      :upload -> 'scp -t'
-      :download -> 'scp -f'
-    end
+  @flags [verbose: '-v', preserve: '-p', recursive: '-r']
 
-    flags = [verbose: '-v', preserve: '-p', recursive: '-r']
+  def build(:upload, path, options) do
+    scp('-t', path, options)
```

Comment:
If you find anything resembling an "official" documentation for these flags, please do drop a link here. To my knowledge, there is no SCP RFC or similar document with a proper specification.

It's basically specified through it's existing implementation(s). Not of great use for documentation purposes though 😬 (http://cvsweb.openbsd.org/cgi-bin/cvsweb/src/usr.bin/ssh/scp.c?rev=1.187&content-type=text/x-cvsweb-markup).

 The links above are the best sources I've come across so far… But I haven't been actively looking for more.